AsteroidTrackingApp/src/main/java/com/viktoriia/utils/ElasticRequests.java

```diff
@@ -31,4 +45,45 @@ public void bulkInsert(BulkRequest bulkRequest) {
             LOGGER.warn(e.getMessage());
         }
     }
+
+    public Asteroid getById(String id) {
+        Asteroid asteroid = null;
+        GetRequest getRequest = new GetRequest(INDEX,id);
+        try {
+            GetResponse getResponse = ElasticsearchConfig.client().get(getRequest, RequestOptions.DEFAULT);
+            JSONObject jsonObject = new JSONObject(getResponse.toString());
+            asteroid = gson.fromJson(jsonObject.toString(), Asteroid.class);
+        } catch (IOException e) {
+            LOGGER.warn(e.getMessage());
+        }
+        return asteroid;
```

Comment:
So, potentially this method could return `null`. What would happen on caller side in that case?
Maybe we could introduce Optional to wrap return value? Or just fail completely?